src/index.html

```diff
@@ -1,17 +1,245 @@
 <!doctype html>
 <html lang="en">
+
   <head>
-    <meta charset="UTF-8">
+    <meta
+      charset="UTF-8"
+    >
```

Comment:
Overall when there is less than 3 attributes in a tag there is no need for spreading it into several lines. We do it when the attribute list is long for readability so this should simply be:
```suggestion
    <meta charset="UTF-8">
```